--- /usr/share/vim/vim80/defaults.vim.orig	2018-03-08 11:01:02.577240651 +0100
+++ /usr/share/vim/vim80/defaults.vim	2018-03-08 11:01:27.715617560 +0100
@@ -66,9 +66,9 @@
 
 " In many terminal emulators the mouse works just fine.  By enabling it you
 " can position the cursor, Visually select and scroll with the mouse.
-if has('mouse')
-  set mouse=a
-endif
+"if has('mouse')
+" set mouse=a
+"endif
 
 " Switch syntax highlighting on when the terminal has colors or when using the
 " GUI (which always has colors).
